### ISSUE TITLE
Have a wrong  after run 'bash run_cocostuff.sh'

### ISSUE THREAD

**Issue by lizhidan** [nightrome/cocostuff10k#5]
Thanks for your code.The following is the result after I implemented the command :bash run_cocostuff.sh.I don't kown where is wrong and how to solve it.Can you help me?

layer {
  name: "drop7_2"
  type: "Dropout"
  bottom: "fc7_2"
  top: "fc7_2"
  dropout_param {
    dropout_ratio: 0.5
  }
}
layer {
  name: "fc8_cocostuff_2"
  type: "Convolution"
  bottom: "fc7_2"
  top: "fc8_cocostuff_2"
  convolution_param {
    num_output: 172
    kernel_size: 1
  }
}
layer {
  name: "fc6_3"
  type: "Convolution"
  bottom: "pool5"
  top: "fc6_3"
  convolution_param {
    num_output: 1024
    pad: 18
    kernel_size: 3
    dilation: 18
  }
}
layer {
  name: "relu6_3"
  type: "ReLU"
  bottom: "fc6_3"
  top: "fc6_3"
}
layer {
  name: "drop6_3"
  type: "Dropout"
  bottom: "fc6_3"
  top: "fc6_3"
  dropout_param {
    dropout_ratio: 0.5
  }
}
layer {
  name: "fc7_3"
  type: "Convolution"
  bottom: "fc6_3"
  top: "fc7_3"
  convolution_param {
    num_output: 1024
    kernel_size: 1
  }
}
layer {
  name: "relu7_3"
  type: "ReLU"
  bottom: "fc7_3"
  top: "fc7_3"
}
layer {
  name: "drop7_3"
  type: "Dropout"
  bottom: "fc7_3"
  top: "fc7_3"
  dropout_param {
    dropout_ratio: 0.5
  }
}
layer {
  name: "fc8_cocostuff_3"
  type: "Convolution"
  bottom: "fc7_3"
  top: "fc8_cocostuff_3"
  convolution_param {
    num_output: 172
    kernel_size: 1
  }
}
layer {
  name: "fc6_4"
  type: "Convolution"
  bottom: "pool5"
  top: "fc6_4"
  convolution_param {
    num_output: 1024
    pad: 24
    kernel_size: 3
    dilation: 24
  }
}
layer {
  name: "relu6_4"
  type: "ReLU"
  bottom: "fc6_4"
  top: "fc6_4"
}
layer {
  name: "drop6_4"
  type: "Dropout"
  bottom: "fc6_4"
  top: "fc6_4"
  dropout_param {
    dropout_ratio: 0.5
  }
}
layer {
  name: "fc7_4"
  type: "Convolution"
  bottom: "fc6_4"
  top: "fc7_4"
  convolution_param {
    num_output: 1024
    kernel_size: 1
  }
}
layer {
  name: "relu7_4"
  type: "ReLU"
  bottom: "fc7_4"
  top: "fc7_4"
}
layer {
  name: "drop7_4"
  type: "Dropout"
  bottom: "fc7_4"
  top: "fc7_4"
  dropout_param {
    dropout_ratio: 0.5
  }
}
layer {
  name: "fc8_cocostuff_4"
  type: "Convolution"
  bottom: "fc7_4"
  top: "fc8_cocostuff_4"
  convolution_param {
    num_output: 172
    kernel_size: 1
  }
}
layer {
  name: "fc8_cocostuff"
  type: "Eltwise"
  bottom: "fc8_cocostuff_1"
  bottom: "fc8_cocostuff_2"
  bottom: "fc8_cocostuff_3"
  bottom: "fc8_cocostuff_4"
  top: "fc8_cocostuff"
  eltwise_param {
    operation: SUM
  }
}
layer {
  name: "fc8_interp"
  type: "Interp"
  bottom: "fc8_cocostuff"
  top: "fc8_interp"
  interp_param {
    zoom_factor: 8
  }
}
layer {
  name: "fc8_mat"
  type: "MatWrite"
  bottom: "fc8_interp"
  include {
    phase: TEST
  }
  mat_write_param {
    prefix: "cocostuff/features/deeplabv2_vgg16/val/fc8/"
    source: "cocostuff/list/val_id.txt"
    strip: 0
    period: 1
  }
}
layer {
  name: "silence"
  type: "Silence"
  bottom: "label"
  bottom: "data_dim"
}
I0616 15:55:01.469554  6570 layer_factory.hpp:77] Creating layer data
I0616 15:55:01.469599  6570 net.cpp:106] Creating Layer data
I0616 15:55:01.469609  6570 net.cpp:411] data -> data
I0616 15:55:01.469641  6570 net.cpp:411] data -> label
I0616 15:55:01.469656  6570 net.cpp:411] data -> data_dim
I0616 15:55:01.469676  6570 image_seg_data_layer.cpp:46] Opening file cocostuff/list/val.txt
I0616 15:55:01.470676  6570 image_seg_data_layer.cpp:68] A total of 1000 images.
F0616 15:55:01.510185  6570 syncedmem.hpp:18] Check failed: error == cudaSuccess (30 vs. 0)  unknown error
*** Check failure stack trace: ***
    @     0x7fbaaa9dadaa  (unknown)
    @     0x7fbaaa9dace4  (unknown)
    @     0x7fbaaa9da6e6  (unknown)
    @     0x7fbaaa9dd687  (unknown)
    @     0x7fbaab106ca8  caffe::SyncedMemory::mutable_cpu_data()
    @     0x7fbaab115f0c  caffe::Blob<>::Reshape()
    @     0x7fbaab1163a9  caffe::Blob<>::Reshape()
    @     0x7fbaab00c95a  caffe::ImageSegDataLayer<>::DataLayerSetUp()
    @     0x7fbaab08c3d3  caffe::ImageDimPrefetchingDataLayer<>::LayerSetUp()
    @     0x7fbaab159005  caffe::Net<>::Init()
    @     0x7fbaab15a758  caffe::Net<>::Net()
    @           0x406bf7  test()
    @           0x4059dc  main
    @     0x7fbaa9ce5f45  (unknown)
    @           0x406111  (unknown)
    @              (nil)  (unknown)
run_cocostuff.sh: line 82:  6570 Aborted                 (core dumped) ${CMD}
hdl1@hdl1:~/cocostuff/models/deeplab-public-ver2$

**Comment by nightrome** [comment]
Hi,
please note that the semantic segmentation model is currently outdated, as it was trained for an older version of COCO-Stuff with 172 classes, instead of 182+1. We are still working on updating that. If you want to try yourself you will change the number 172 in the script above and finetune the model again.
